android/app/src/test/java/network/mysterium/ui/TokenModelTest.kt

```diff
@@ -0,0 +1,22 @@
+package network.mysterium.ui
+
+import org.junit.Assert.*
+import org.junit.Test
+
+class TokenModelTest {
```

Comment:
I suggest taking a look into [speck](https://www.spekframework.org/) (spockframework adaptation for Kotlin). But if you're stuck with old junit then you could at least do parameterization:

```kt
@RunWith(Parameterized::class)
class TokenModelTest(private val token: Long, private val display: String) {

    companion object {
        @JvmStatic
        @Parameterized.Parameters
        fun data(): Collection<Array<Any>> {
            return listOf(
                    arrayOf(124_00_000_000L, "124.000 MYSTT"),
                    arrayOf(124_54_000_000L, "124.540 MYSTT"),
                    arrayOf(114_34_000_000L, "114.340 MYSTT"),
                    arrayOf(123_01_000_000L, "123.010 MYSTT")
            )
        }
    }
    @Test
    fun testDisplayValueFormatting() {
        val model = TokenModel(token = token)
        assertEquals(display, model.displayValue)
    }
}
```